automates/text_reading/webapp/app/controllers/HomeController.scala

```diff
@@ -248,7 +248,7 @@ class HomeController @Inject()(cc: ControllerComponents) extends AbstractControl
       val blockIdx = location.last
 
       for (m <- menInTextBlocks) {
-        val newMen = m.withAttachment(new MentionLocationAttachment(pageNum, blockIdx, "MentionLocation"))
+        val newMen = m.withAttachment(new MentionLocationAttachment("Fixme: pass filename", pageNum, blockIdx, "MentionLocation"))
```

Comment:
What is that "Fixme: pass filename" for? Is that a message shown when there's some kind of problem with the file name??

Reply:
argh I forgot about that one. I am having trouble fixing it. I will try before I request re-review. 

Basically, I am now storing filename in attachments and I am having trouble and I am having trouble accessing filename in this method.

Reply:
Okay, please let me know if you need any help with this!